app/controllers/benchmark_runs_controller.rb

```diff
@@ -27,10 +27,22 @@ def create
       benchmark_result_type: benchmark_result_type
     )
 
+    if params[:commit_hash]
+      commit_objects = BenchmarkRun.where( initiator: initiator, benchmark_type: benchmark_type,
+                       benchmark_result_type: benchmark_result_type).limit(10)
+      commit_results = Array.new
+      commit_objects.each do |object|
+        commit_results << object.result[category]
+      end
+    end
```

Comment:
The background job can be called after the current `benchmark_run` is saved to the database successfully.